test/cluster/test_tablets.py

```diff
@@ -231,6 +231,41 @@ async def test_tablet_mutation_fragments_unowned_partition(manager: ManagerClien
                 await cql.run_async(f"SELECT partition_region FROM MUTATION_FRAGMENTS({ks}.test) WHERE pk={k}", host=host[0])
 
 
+# The test checks that describe_ring and range_to_address_map API return
+# information that's consistent with system.tablets contents
+@pytest.mark.parametrize("endpoint", ["describe_ring", "range_to_endpoint"])
+@pytest.mark.asyncio
+async def test_tablets_api_consistency(manager: ManagerClient, endpoint):
+    servers = await manager.servers_add(3, config={"rf_rack_valid_keyspaces": "false", "enable_tablets": "true"})
+    hosts = { await manager.get_host_id(s.server_id): s.ip_addr for s in servers }
+    cql = manager.get_cql()
+    async with new_test_keyspace(manager, "WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 2} and tablets = {'initial': 4}") as ks:
```

Comment:
Soon with rf-rack-list we would want this configuration to use a specified subset of 2 out of 3 racks (if not given explicitly, it will be automatically chosen when expanding `'replication_factor': 2` into a rack list, so I;m not sure it's an interesting case to test.  Why not go with `"rf_rack_valid_keyspaces": "true"` and `'replication_factor': 3}`

Reply:
That's what I meant, that with rf rack list the replica set would be the same for all ranges anyhow, just limited to a subset of the racks. If you create a cluster with 3 racks, but 2 or more nodes per rack, the replica set should be different for different tablets on the per-node basis (not per rack).

Reply:
OK